packages/api/src/idpuser/models.ts

```diff
@@ -0,0 +1,11 @@
+/** @format */
+
+export interface TwoFactorRequest {
+  emailFromAuthToken: string;
+}
```

Comment:
This appears to only be used in tests. I would either move its definition to the test file, or just delete it and use `{emailFromAuthToken: string}` in its place in the tests.